### DISCUSSION TITLE
Support TRON in ABI functions

### DISCUSSION THREAD

**Discussion by netanel-utila** [wevm/viem#3143] [General]
Hi,

I'm using the library for EVM and it works great, but now i need to support TRON and I have issues with the ABI usage, for example:

Address "TE3PsmHzHUCXEQQYc3v5HLZyYKsiYkdXoq" is invalid

Is there a way to use viem with TRON? The ABI should be compatible with EVM

Thanks

**Comment by jxom** [comment]
Viem only supports EVM addresses (40 char hexadecimal).

**Reply by netanel-utila** [reply]
So we can't use viem with TRON?

**Reply by jxom** [reply]
Needs to be an EVM chain.
